docs/understanding-tracking-design/managing-tracking-scenarios-api/index.md

```diff
@@ -0,0 +1,367 @@
+```
+
+- **Name**: It validates that the `scenario.name` of a tracking scenario is unique within the data structure inferred from the `scenario.event.source`.
+- **Entities**: If present during creation, it will validate that the entities, `scenario.entities.tracked` and `scenario.entities.enriched`, are not duplicated.
+
+### Compatibility Check
+
+If a `scenario.event` is provided during creation, a compatibility check will be performed between the provided event and the one referenced in Iglu by `scenario.event.source`.
+
+This check is performed for the version indicated by the Iglu URI, referred to as the **current** version. Additionally, the compatibility check is also run for the **latest** version. The compatibility check results in three possible values: **Compatible**, **SchemaIncompatible**, or **SchemaUndecidable**. The different errors will be provided in the `errors` field of the [response](#response-format).
+
+- If **Compatible**, no errors or warnings will be attached to the `errors` response field.
+- If **SchemaIncompatible**, an error will be attached to the `errors` response field when the **current** version is incompatible, resulting in a **422 Unprocessable Entity** response without being able to create the tracking scenario. If the **latest** version is incompatible, a warning will be attached to the `errors` field, but a **201 Created** response will still be returned.
+- If **SchemaUndecidable**, a warning will be attached to the `errors` response field.
+
+## Editing a Tracking Scenario
+
+Use this request to edit a tracking scenario within an organization. The [request and response](#request-and-response) follow the same format as during creation.
+
+`PUT /api/msc/v1/organizations/{organizationId}/tracking-scenarios/v1/{trackingScenarioId}`
+
+### Publishing a Tracking Scenario
+
+By default, when a tracking scenario is created and no value is provided for `scenario.status`, it is set to **draft**. This suggests a tracking scenario lifecycle that we recommend following, especially when using the API in conjunction with the Console.
+
+```mermaid
+graph LR
+    style Start fill:#633EB5, stroke:#000000, stroke-width:0px;
+    style Draft color:#724512, fill:#FEEEBD, stroke:#724512, stroke-width:1px; 
+    style Published color:#3F2874, fill:#F0EBF8, stroke:#3F2874, stroke-width:1px;
+    style Deprecated color:#89251F, fill:#FDF3F2, stroke:#89251F, stroke-width:1px;
+    style Deleted fill:#D63A31, stroke:#000000, stroke-width:1px;
+    linkStyle default stroke-width:2px,fill:#F2F4F7,stroke:#633EB5,color:#633EB5
+
+    Start(( )) -->|Create| Draft
+    Draft -->|Publish| Published
+    Draft -->|Delete| Deleted
+    Published -->|Deprecate| Deprecated
+    Deprecated -->|Undeprecate| Draft
+    Deleted((Deleted))
+```
+
+In addition to this lifecycle and in conjunction with versioning, we enforce that when a tracking scenario is **published**, the versions between two published versions are **squashed**. For example:
```

Comment:
I think I would call this “discarded” rather than “squashed”...

Reply:
Replaced in favor of "discarded".